flight_backend/api/tasks.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, date, timezone
+
+from celery import shared_task
+from celery.schedules import crontab
+from celery.decorators import periodic_task
+
+from django.conf import settings
+from django.shortcuts import get_object_or_404
+from django.core.mail import send_mail
+from django.template.loader import get_template
+
+from api.models import Ticket
+from api.constants import BOOKED, RESERVED
+
+
+@shared_task()
+def flight_booking_notification(pk):
+    ticket = get_object_or_404(Ticket.objects.all(), pk=pk)
```

Comment:
how many db calls does this make? (compared using `.get()` with `try/except`)

Reply:
It makes a db call once using the pk as a param, if no ticket exist with such pk, it returns 404.